src/M2ulPhyS.cpp

```diff
@@ -104,7 +104,29 @@ void M2ulPhyS::initVariables() {
     // read serial mesh and corresponding partition file (the hdf file that was generated
     // when starting from scratch).
 
-    serial_mesh = new Mesh(config.GetMeshFileName().c_str());
+    // w/o ini periodic option
+    // serial_mesh = new Mesh(config.GetMeshFileName().c_str());
+
+    // periodic treatments
```

Comment:
Is it possible to refactor so that the code for the periodic modification is not duplicated?

Reply:
yes, done in da32e94